packages/special-pages/pages/new-tab/src/js/mock-transport.js

```diff
@@ -0,0 +1,103 @@
+import { TestTransportConfig } from '@duckduckgo/messaging'
+
+export function mockTransport () {
```

Comment:
the NTP is complicated, and to work on it, we need more complex mocks.

This can be used in local development, and is stripped out for production